pilot/pkg/config/kube/crdclient/metrics.go

```diff
@@ -19,20 +19,15 @@ import (
 )
 
 var (
-	typeTag  = monitoring.MustCreateLabel("type")
-	eventTag = monitoring.MustCreateLabel("event")
+	typeTag  = monitoring.CreateLabel("type")
```

Comment:
Not entirely sure why this was renamed - either remove or leave it as is, we can live with 'Must' for a while...